doc/040_backup.rst

```diff
@@ -93,7 +93,7 @@ Basic example:
     *.go
     # exclude foo/x/y/z/bar foo/x/bar foo/bar
     foo/**/bar
-    $ restic -r /tmp/backup backup ~/work --exclude=*.c --exclude-file=exclude
+    $ restic -r /tmp/backup backup ~/work --exclude="*.c" --exclude-file=exclude
```

Comment:
For the record: This is not strictly necessary, but I think it's a good idea. Shells will resolve `--exclude *.c`, but won't resolve `--exclude=*.c`.